src/migrations/20240626135812-clean-up-tr-goal-groups.js

```diff
@@ -0,0 +1,52 @@
+const {
+  prepMigration,
+} = require('../lib/migration');
+
+module.exports = {
```

Comment:
You could just increment the version number of the goal similarity groups instead of doing this, that is what it's for

Reply:
oops ok Ill just up the version instead.